samples/detect.v1p3beta1.js

```diff
@@ -0,0 +1,175 @@
+
+  client
+    .objectLocalization(uri)
+    .then(results => {
+      console.log(JSON.stringify(results, null, '\t'));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END Object Localization ]
+}
+
+function detectHandwritingOCR(file) {
+  // [START vision detectHandwritingOCR]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p3beta1;
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const file = 'Local image file, e.g. /path/to/image.png';
+
+  client
+    .documentTextDetection(file)
+    .then(results => {
+      const fullTextAnnotation = results[0].fullTextAnnotation;
+      console.log(`Full text: ${fullTextAnnotation.text}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END vision detectHandwritingOCR]
+}
+
+function detectHandwritingGCS(uri) {
+  // [START vision detectHandwritingOCRGCS]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p3beta1;
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
+
+  client
+    .documentTextDetection(uri)
+    .then(results => {
+      const fullTextAnnotation = results[0].fullTextAnnotation;
+      console.log(`Full text: ${fullTextAnnotation.text}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END vision detectHandwritingOCRGCS]
+}
+
+require(`yargs`)
+  .demand(1)
+  .command(
+    `localizeObjects`,
+    `Detects Objects in a local image file`,
+    {},
+    opts => localizeObjects(opts.fileName)
+  )
+  .command(
+    `localizeObjectsGCS`,
+    `Detects Objects Google Cloud Storage Bucket`,
+    {},
+    opts => localizeObjectsGCS(opts.gcsUri)
+  )
+  .command(
+    `detectHandwriting`,
+    `Detects handwriting in a local image file.`,
+    {},
+    opts => detectHandwritingOCR(opts.handwritingSample)
+  )
+  .command(
+    `detectHandwritingGCS`,
+    `Detects handwriting from Google Cloud Storage Bucket.`,
+    {},
+    opts => detectHandwritingGCS(opts.handwritingSample)
+  )
+  .options({
+    fileName: {
+      alias: 'f',
+      default: `./resources/duck_and_truck.jpg`,
+      global: true,
+      requiresArg: true,
+      type: 'string',
+    },
+    gcsUri: {
+      alias: 'u',
+      global: true,
+      requiresArg: true,
+      type: 'string',
+    },
+    handwritingSample: {
+      alias: 'h',
+      default: `./resources/handwritten.jpg`,
+      global: true,
+      requiresArg: true,
+      type: 'string',
+    },
+    handwritingGcsUri: {
+      alias: 'u',
+      default: `gs://nodejs-docs-samples//handwritten.jpg`,
+      global: true,
+      requiresArg: true,
+      type: 'string',
+    },
+  })
+  .example(`node $0 localizeObjects -f ./resources/duck_and_truck.jpg`)
+  .example(`node $0 localizeObjectsGCS gcsuri`)
```

Comment:
#nit I vote `gs://` something rather than `gcsuri`

Maybe:

```js
.example(`node $0 localizeObjectsGCS gs://my-bucket/image.jpg`)
```

Gives a clue of what the argument should look like :)